### PR TITLE
update for gulp 4

### PR DESCRIPTION
**Pull request by adrianjost**
I think it is working now, with the fix from: https://github.com/gtg092x/gulp-sftp/issues/78#issuecomment-356475605

- [x] https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 'use strict';
 var clearDirectory = require('./clear').clearDirectory;
+var log = require('fancy-log');
```
Comment (krysalead) [inline index.js]: Can you please name it the same way in both files?

File: index.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+var colors = require('ansi-colors');
+var PluginError = require('plugin-error');
```
Comment (krysalead) [inline index.js:5]: Is this plugin error working with gulp 3?

File: index.js (hunk 1 above)
Comment (adrianjost) [reply index.js]: hi, sorry for the really delayed answer. 
I renamed it now. 

I wondered about this difference as well during implementing it, but it wasn't named the same before, so I continued naming it differently.

File: index.js (hunk 2 above)
Comment (adrianjost) [reply index.js:5]: I don't know for sure, but I guess it is. The `plugin-error` package was extracted from gulp3 into a seperate project. The new package doesn't has any dependencies to gulp 3 so I guess it's just simply implemented twice if you use gulp 3, but this shoudn't be an issue.

**Pull request closed without merging**